eclipse-scout-core/src/desktop/hybrid/HybridManagerAdapter.ts

```diff
@@ -39,12 +40,111 @@ export class HybridManagerAdapter extends ModelAdapter {
   }
 
   protected _onWidgetHybridAction(event: HybridActionEvent) {
-    this._send('hybridAction', event.data);
+    this._send('hybridAction', {
+      actionType: event.data.actionType, // add as first property (devtools sometimes show properties in that order)
+      id: event.data.id,
+      data: event.data.data,
+      contextElements: this._contextElementsToJson(event.data.contextElements) || undefined
```

Comment:
Not technically necessary, but helpful when inspecting REST traffic, because it reduces unnecessary "noise" on the transport channel. Undefined values are automatically omitted during JSON serialization.